src/manager.cpp

```diff
@@ -327,7 +327,7 @@ static std::map<pid_t, std::string> start_modules(Config& config, MQTTAbstractio
         // FIXME (aw): implicitely adding ModuleReadyInfo and setting its ready member
         auto module_it = modules_ready.emplace(module_name, ModuleReadyInfo{false, nullptr}).first;
 
-        Handler module_ready_handler = [module_name, &mqtt_abstraction](nlohmann::json json) {
+        Handler module_ready_handler = [module_name, &mqtt_abstraction, &rs](nlohmann::json json) {
```

Comment:
be careful here, that rs doesn't get out of scope, furthermore, only rs.mqtt_everest_prefix is needed, this could be captured as well